finetune/inference/arc_infer.py

```
@@ -0,0 +1,155 @@
+                correct_counter += 1
+                correct_task.append(d['uid'])
+                break
+            else:
+                # breakpoint()
+                print("Incorrect!")
+        else:
+            print("Wrong output format")
+
+    with open(saving_file, "w") as f:
+        f.write("\n".join(json.dumps(p) for p in all_responses))
+  except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)